src/constants/productTokens.ts

```diff
@@ -9,7 +9,7 @@ export interface ProductToken {
   image: string
   coingeckoId: string
   tokensetsId: string
-  fees: any | undefined
+  fees: { streamingFee: string } | undefined
```

Comment:
Proposal for a refactor that is out of scope for this PR:
Modify `IndexToken` to not be of type `ProductToken` but instead pick a subset of props (all props excluding `fees`) b/c all other product tokens have `fees`. Then we can make this field:

```typescript
  fees: { streamingFee: string }
```

Reply:
I like this idea. There are some other small bits with how it's currently set up I would also like to change. We may need to discuss this soon.